File: dotnet/System/Workspace/Adapters/Allors.Workspace.Adapters.Json/Api/Invoke/InvokeResult.cs
// <copyright file="RemoteInvokeResult.cs" company="Allors bvba">
// Copyright (c) Allors bvba. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Json
{
    using Allors.Protocol.Json.Api.Invoke;
    using Allors.Workspace.Response;

    public class InvokeResult : Result, IInvokeResult
    {
        public InvokeResult(Workspace workspace, InvokeResponse invokeResponse) : base(workspace, invokeResponse)
        {
        }
    }
}

File: typescript/workspace/system/adapters-json/src/lib/database/invoke/invoke-result.ts
import { IInvokeResult, ISession } from '@allors/workspace-system-domain';
import { InvokeResponse } from '@allors/database-system-protocol-json';
import { Result } from '../result';

export class InvokeResult extends Result implements IInvokeResult {
  constructor(session: ISession, invokeResponse: InvokeResponse) {
    super(session, invokeResponse);
  }
}
